src/Provider/GitLab.php

```diff
@@ -62,4 +63,14 @@ public function generatePullRequestLink(string $package, int $pr) : string
 
         return sprintf('https://gitlab.com/%s/merge_requests/%d', $package, $pr);
     }
+
+    public function getName() : string
+    {
+        return 'GitLab';
+    }
+
+    public function getDomainName() : string
+    {
+        return 'gitlab.com';
```

Comment:
May be make it configurable rather than hardcoded, to allow for self-hosted gitlab?

Reply:
Users can create a custom provider in such scenarios.

Reply:
Hm... they can't, actually. The `GetConfigValuesTrait` hardcodes that information. The configuration would need to be able to inject the domain.

I'll go ahead and merge now, and then see if I can address that issue before I do a release.